packages/babel-plugin-transform-es2015-classes/src/index.js

```diff
@@ -46,6 +46,10 @@ export default function ({ types: t }) {
         if (state.opts.loose) Constructor = LooseTransformer;
 
         path.replaceWith(new Constructor(path, state.file).run());
+
+        if (path.isCallExpression() && path.get("callee").isArrowFunctionExpression()) {
+          path.get("callee").arrowFunctionToExpression();
```

Comment:
Yeah, we can always drop this now if we want, since the plugins are already in a fine order for it AFAIK, I just figured it would be better to leave that for a separate PR.